addon/webextension/background/analytics.js

```diff
@@ -94,6 +94,17 @@ this.analytics = (function() {
       label = undefined;
     }
     options = options || {};
+
+    // Don't send events if in private browsing.
+    if (options.hasOwnProperty('incognito')) {
```

Comment:
What's the reason for `hasOwnProperty` here?

Reply:
You could do `options.incognito !== undefined` – but I think simply testing for truthiness would do what's needed here?

Reply:
If it's falsey because it has a value of `false`, then `delete` below won't happen.  I suppose I could call the `delete` without a check since that won't throw an error.